src/pages/UnlockedWallet/Layout.tsx

```diff
@@ -36,13 +36,13 @@ import { appHeaderHeightPx, walletSidebarWidthPx } from '@/style/globalStyles'
 import { TxType } from '@/types/transactions'
```

Comment:
Let's call this file `UnlockedWalletLayout` for consistency and "searchability" sake? :)